Automattic/wp-calypso
# Use Inter and SF Pro Text fonts for Blaze Pro's authentication

Related to A8C-DSP's issue 1824

## Proposed Changes
- Start using Inter font for headings and some signs,
- Use SF Pro Text for the rest of signs. 

## Why are these changes being made?
We want to be compliant with [Figma](https://www.figma.com/design/0YQJMNMOVVOMqzCWufls0l/BlazePro-(Self-Served-for-Advertisers)?node-id=7096-11597&t=Z2WDCMO4snOmiMA8-0) designs from Bart.

## Testing Instructions
- Use the scripts described in https://github.com/Automattic/wp-calypso/pull/91095
- Generate 2 links for log in and sign up
- Open WordPress.com and make sure your test account has 2FA enabled
- Run Calypso locally or use a live link 
- Compare what you see in Figma with that you get in Calypso (actual fonts should match with designs)
- Open a Sign Up URL:

![Screenshot 2024-06-10 at 15 54 34](https://github.com/Automattic/wp-calypso/assets/115007291/38d6727e-c046-444e-a4e4-104e912160e1)

- Open Log In URL
![Screenshot 2024-06-10 at 15 51 49](https://github.com/Automattic/wp-calypso/assets/115007291/cd8db713-a3bf-46cc-bb5f-7784f9e857b1)

- Enter email and password to Log in,
- Make sure 2FA designed accordingly,

![Screenshot 2024-06-10 at 15 56 27](https://github.com/Automattic/wp-calypso/assets/115007291/d36d5304-5e0a-48a1-a657-de1f327668d7)

- Open a page to login using a Backup Key.

![Screenshot 2024-06-10 at 15 58 16](https://github.com/Automattic/wp-calypso/assets/115007291/487f6ff2-e2e8-4595-81c8-1731bb12751b)

- For testing processing screen, there's an option to use a Live Link, but in that case you have to create a new account. It's a bit easier to use a local Calypso and adjust the line `847` in `client/signup/main.jsx` ([here](https://github.com/Automattic/wp-calypso/blob/trunk/client/signup/main.jsx#L847)):

```js 
// Change the line 847 to become:
{ true || this.state.shouldShowLoadingScreen ? (
```

- Open a Sign Up URL (generated with the script)
- Make sure the fonts are as in designs

![Screenshot 2024-06-10 at 15 59 46](https://github.com/Automattic/wp-calypso/assets/115007291/ee30cacb-2409-4cf5-aa95-68125bb4d863)

## Pre-merge Checklist
